src/Paket.Core/Files/ProjectFile.fs

```diff
@@ -1079,7 +1079,7 @@ module ProjectFile =
 
             let importTargets = defaultArg installSettings.ImportTargets true
             
-            let allFrameworks = applyRestrictionsToTargets ((thirdOf3 kv.Value)) KnownTargetProfiles.AllProfiles
+            let allFrameworks = applyRestrictionsToTargets ((thirdOf3 kv.Value)) (KnownTargetProfiles.AllDotNetStandardProfiles @ KnownTargetProfiles.AllDotNetProfiles)
```

Comment:
PLEASE REVIEW: I think we don't care about runtimes when deciding whether to install a .targets / .props, is that correct?